### PR TITLE
Upgrade to iOS 14 and Xcode 12

### PR DESCRIPTION
**Pull request by gr0**
Hey @amir-hadzic!

have a look at this PR. First of all it upgrades the code to iOS 14 and Xcode 12. The build is passing on [Travis](https://travis-ci.org/github/sematext/sematext-logsene-ios/builds/734988827), so everything should be OK. 

The major change in this PR is the removal of the `SQLLite.swift` as it is no longer maintained. Our users were also mentioning that they would like to get a dependency free library, so that they are not forced to yet another dependency in the project.

### CODE REVIEW

File: Example/Logsene/AppDelegate.swift [hunk 1]
```diff
@@ -34,5 +34,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func applicationWillTerminate(_ application: UIApplication) {
+        LogseneTerminate()
```
Comment (amir-hadzic) [inline Example/Logsene/AppDelegate.swift]: Should we mention this in the README?

File: Package.resolved [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "SQLite.swift",
+        "repositoryURL": "https://github.com/stephencelis/SQLite.swift.git",
```
Comment (amir-hadzic) [inline Package.resolved]: Is this auto generated, do we need if we are removing SQLite.swift from the lib?

**Review by amir-hadzic — APPROVED** [review]

File: Package.resolved (hunk 2 above)
Comment (gr0) [reply Package.resolved]: Ahh I will remove that, it was automatically generated.

File: Example/Logsene/AppDelegate.swift (hunk 1 above)
Comment (gr0) [reply Example/Logsene/AppDelegate.swift]: This is a left over, no longer needed. I will remove that.

**Pull request merged**